### PR TITLE
update intro

### PR DESCRIPTION
**Pull request by ryanatkn**

### CODE REVIEW

File: src/lib/Intro.svelte [hunk 1]
```diff
@@ -1,22 +1,19 @@
 <p class="panel padded-xl">
 	<strong>Felt.social</strong>
-	is a
-	<small><code>work in progress</code></small>
-	community, service, and business powered by
+	is a community, service, and business powered by
 	<a href="https://www.felt.dev/">Felt</a>, a free and open-source toolkit for building and
-	maintaing communities. You can learn more about Felt
-	<a href="https://www.felt.dev/about">here</a>.
```
Comment (ryanatkn) [inline src/lib/Intro.svelte:8]: not about deleting that last line, trying it this way to see if simpler is better

File: src/lib/Intro.svelte [hunk 2]
```diff
@@ -1,22 +1,19 @@
...
 	<strong>Felt.social</strong>
-	is a
-	<small><code>work in progress</code></small>
-	community, service, and business powered by
-	<a href="https://www.felt.dev/">Felt</a>, a free and open-source toolkit for building and
-	maintaing communities. You can learn more about Felt
-	<a href="https://www.felt.dev/about">here</a>.
+	is a community, service, and business powered by
+	<a href="https://www.felt.dev/">Felt</a>, a free and open source toolkit for building and
+	maintaining communities.
```
Comment (ryanatkn) [inline src/lib/Intro.svelte:5]: typo `maintaing` -> `maintaining`

File: src/lib/Intro.svelte [hunk 3]
```diff
@@ -1,22 +1,19 @@
...
 </p>
 <p class="panel padded-xl">
-	In our mission to make our lives better with technology, we are building on Felt to provide a
+	In our mission to make our lives better with technology, we're building on Felt to provide a
```
Comment (ryanatkn) [inline src/lib/Intro.svelte:8]: this felt stilted to me, still seems like something could be improved

File: src/lib/Intro.svelte [hunk 4]
```diff
@@ -1,22 +1,19 @@
 <p class="panel padded-xl">
 	<strong>Felt.social</strong>
-	is a
-	<small><code>work in progress</code></small>
```
Comment (ryanatkn) [inline src/lib/Intro.svelte:4]: might as well remove this now

**Review by greatbacon — APPROVED** [review]

**Pull request merged**